docs/velocity/admin/reference/configuration.md

```diff
@@ -18,10 +18,9 @@ There are a few "special" data types in the Velocity configuration.
 
 ### Chat
 
-Chat messages may be provided in legacy color code format or in JSON format.
+Chat messages may be provided in [MiniMessage](https://docs.advntr.dev/minimessage/format.html) format.
 
-RGB support (using the `&#rrggbb` format) is available and JSON messages are deserialized for
-Minecraft 1.16.
+RGB support (using the `<#rrggbb>` format) is available for Minecraft 1.16.
```

Comment:
Should this specify the format here? Or just simply say RGB support is available for Minecraft 1.16+?

I feel like the format was specifically mentioned because it differed for different applications but minimessage is all standardized.